gen_cert.py

```diff
@@ -1878,10 +1886,11 @@ def fontlist_with_style(a_style):
 
         # SECTION: Extra achievements
         achievements_string = ""
-        achievements_description_string = self.interstitial_texts[grade]
+        achievements_description_string = self.interstitial_texts[grade].decode('utf-8')
         if grade and grade.lower() != 'pass':
-            achievements_string = "with <b>{0}</b>.<br /><br />".format(grade)
-        achievements_paragraph = u"{0}{1}".format(achievements_string, achievements_description_string)
+            grade_html = u"<b>{0}</b>".format(grade.decode('utf-8'))
```

Comment:
Yeah, but "previous ones use" isn't a good excuse here because they previously used positionals _everywhere_, even when there were multiple (look 2 lines down). This was written before edX adopted this style so they often used positionals.  At this point, we should also use kwargs for string formatting.